lib/rspec/acts_as_matchers.rb

```diff
@@ -0,0 +1,7 @@
+RSpec::Matchers.define :acts_as do |actable|
```

Comment:
You're right! I was thinking like "an object that acts as", but I think the proper phrasing would be "an object that would act as" :)